### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by Markolie**
Fixes https://github.com/ParadiseSS13/Paradise/issues/6365
Looks like I forgot to update the function parameters in the goon_powers file. This fixes some genetics spells not working.

Fixes https://github.com/ParadiseSS13/Paradise/issues/6331
Posibrains can no longer whisper when silenced.

Fixes https://github.com/ParadiseSS13/Paradise/issues/6375
pAI's now have support for factions: the faction variable on the card is copied to the pAI mob. The pAI card on the Syndicate outpost now has the Syndicate faction.

Probably fixes https://github.com/ParadiseSS13/Paradise/issues/6384
goon_disabilities.dm was encoded in ANSI rather than UTF-8. I think that's what's causing certain special characters to disappear from chat.

Fixes https://github.com/ParadiseSS13/Paradise/issues/6378
Mouthless humans can no longer eat crayons. Also added an eating sound to the crayon eating.

Fixes https://github.com/ParadiseSS13/Paradise/issues/6371
You can now use instruments while buckled.

Fixes https://github.com/ParadiseSS13/Paradise/issues/6370
Emagging one fax machine will no longer give other machines access to the Syndicate destination. Also fixes the "send" envelope icon in the interface.

Fixes https://github.com/ParadiseSS13/Paradise/issues/6389
Evidence bags will now always show the item inside of them.

Fixes https://github.com/ParadiseSS13/Paradise/issues/6399
Looks like the ion trail effect uses the "blank" icon_state to blank out the icon, but blank was now an actual icon.

🆑 Markolie
fix: Fixed a number of genetics powers not working.
fix: Posibrains can no longer whisper when silenced.
fix: Syndicate turrets will no longer target the Syndicate pAI found on their outpost.
fix: Fixed the "Swedish" disability occasionally hiding parts of speech.
fix: Mouthless players can no longer eat crayons (such as IPC's).
fix: Instruments can now be interacted with while buckled.
fix: Emagging a single fax machine will no longer give all fax machines access to the Syndicate destination.
fix: Evidence bags will now always show the item inside of them.
fix: Spacepods leaving a trail of humans instead of ion trail effects.
/🆑

### CODE REVIEW

**Comment by Spacemanspark** [comment]
Neat.

File: code/modules/mob/living/carbon/brain/say.dm [hunk 1]
```diff
@@ -1,23 +1,35 @@
...
+
+	..()
+	
+/mob/living/carbon/brain/can_speak(var/warning = FALSE)
+	. = ..()
 
-		..(message)
+	if(!container)
+		. = FALSE
+	else if(!istype(container, /obj/item/device/mmi))
```
Comment (tigercat2000) [inline code/modules/mob/living/carbon/brain/say.dm]: This could probably be merged with the first `if`
`if(!istype(container, /obj/item/device/mmi))`

**Comment by SamHPurp** [comment]
I appreciate you, @Markolie

**Comment by Markolie** [comment]
Thank you :).

**Comment by Davdichan** [comment]
*Evidence bag, not detective bag

**Comment by Markolie** [comment]
Thanks - corrected.

**Comment by Markolie** [comment]
This is ready to go @Crazylemon64 @Fox-McCloud

**Comment by scrubmcnoob** [comment]
@Markolie For this

> Fixes #6399
Looks like the ion trail effect uses the "blank" icon_state to blank out the icon, but blank was now an actual icon.

Does it also fix jetpacks and pretty much any propulsion item because they also leave behind the black man instead of an ion trail?

**Comment by Markolie** [comment]
Yup, it fixes everything that uses that ion trail effect.

**Pull request merged**